src/main/java/stream/flarebot/flarebot/util/pagination/PagedEmbedBuilder.java

```diff
@@ -65,13 +68,34 @@ public PagedEmbed build() {
         boolean pageCounts = false;
         if (list.getPages() > 1)
             pageCounts = true;
-        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color);
+        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color, groupPrefix, groupsPerPage);
     }
 
     public void setColor(Color color) {
         this.color = color;
     }
 
+    /**
+     * Set the embed to use the groups within the {@link PaginationList}.
+     */
+    public void useGroups() {
+        this.groupPrefix = "";
+    }
+
+    public void useGroups(int groupsPerPage) {
```

Comment:
Document

Reply:
Also this should just call useGroups(groupsPerPage), no point have a global variable with a default if it isn't used.